src/track/beats.cpp

```diff
@@ -14,6 +14,11 @@ namespace {
 inline bool BeatLessThan(const track::io::Beat& beat1, const track::io::Beat& beat2) {
     return beat1.frame_position() < beat2.frame_position();
 }
+inline bool TimeSignatureMarkerLessThan(
```

Comment:
```suggestion
inline bool TimeSignatureMarkerEarlier(
```
This clarifies that it is the positions of the markers that is being compared, not what they mark.